app/src/main/java/com/cornellappdev/transit/ui/screens/RouteScreen.kt

```diff
@@ -0,0 +1,164 @@
+import com.cornellappdev.transit.ui.theme.PrimaryText
+import com.cornellappdev.transit.ui.theme.SecondaryText
+import com.cornellappdev.transit.ui.theme.sfProDisplayFamily
+import com.cornellappdev.transit.ui.viewmodels.HomeViewModel
+import com.google.accompanist.permissions.ExperimentalPermissionsApi
+
+@OptIn(ExperimentalPermissionsApi::class, ExperimentalMaterial3Api::class)
+@Composable
+fun RouteScreen(
+    //homeViewModel: HomeViewModel = hiltViewModel()
+) {
+
+    Column(modifier = Modifier.fillMaxSize()) {
+        //TODO make an AppBarColors class w/ the right colors and correct icon
+        TopAppBar(
+            title = { Text(text = "Route Options", fontFamily = sfProDisplayFamily, fontStyle = FontStyle.Normal)},
+            navigationIcon = {
+                IconButton(onClick = { /*TODO make this button nav back to whatever screen*/ }) {
+                    Icon(
+                        imageVector = Icons.Outlined.KeyboardArrowLeft,
+                        contentDescription = ""
+                    )
+                }
+            }
+
+        )
+
+        Divider(thickness = 1.dp, color = DividerGrey)
+        Row(
+            modifier = Modifier
+                .padding(horizontal = 20.dp, vertical = 12.dp),
+            horizontalArrangement = Arrangement.spacedBy(16.dp),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            // From-to column
+            Column(verticalArrangement = Arrangement.spacedBy(24.dp)) {
+                Text(
+                    text = "From",
+                    color = PrimaryText,
+                    fontFamily = sfProDisplayFamily,
+                    fontWeight = FontWeight.Normal
+                )
+                Text(
+                    text = "To",
+                    color = PrimaryText,
+                    fontFamily = sfProDisplayFamily,
+                    fontWeight = FontWeight.Normal
+                )
+            }
+
+            Icon(
+                painter = painterResource(id = R.drawable.route_icon),
+                contentDescription = "",
+                modifier = Modifier.size(width = 16.dp, height = 54.dp),
+                tint = Color.Unspecified
+            )
+
+            //TODO user should be able to enter location/destination
+            Column(verticalArrangement = Arrangement.spacedBy(12.dp)) {
+                Box(
+                    modifier = Modifier
+                        .background(color = DividerGrey, shape = RoundedCornerShape(8.dp))
+                        .fillMaxWidth(0.9f)
+                ) {
+                    Text(
+                        text = "Current Location",
+                        modifier = Modifier.padding(horizontal = 12.dp, vertical = 6.dp),
+                        color = PrimaryText,
+                        fontFamily = sfProDisplayFamily,
+                        fontWeight = FontWeight.Normal
+                    )
```

Comment:
Ask Jonathan about making the viewmodel for this screen--that should be your next task. There, you'll make flows to specify what text to show here. For now, though, this placeholder is good!